public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+To start, try modeling function using geometry.
+
+<Figure
+  image="figures/12.34.svg"
+  caption="Unlike in previous problems, $dx$ does not represent geometric length, but instead represents **area**."
+/>
+
+<Question
+  question="What is the derivative of the function $f(x) = \sqrt{x}$?"
+  choice1="$\sqrt{2}x$"
+  choice2="$\frac{1}{2\sqrt{x}}$"
+  choice3="$\frac{x}{\sqrt{2}}$"
+  choice4="$\frac{1}{2}x$"
+  answer={2}
+>
+
+In order to find the derivative of the function $f(x) = \sqrt{x}$ we can model its geometry as the area of the square, where given the area $x$ the function returns the length of the square's side which is the square root of $x$.
+
+$$
+x = \sqrt{x} \sqrt{x} \\
+$$
+
+Then, we want to find $d\sqrt{x}$, the change of the function's output, when the input $x$ is nudged by a small value $dx$. Since $x$ represents area, it makes sense that a small nudge $dx$ should also be area! 
+
+<Figure
+  image="figures/12.34-explanation.svg"
+  caption="Unlike in previous problems, $dx$ is not geometric length, but instead represents **area**."
+/>
+
+The added area changes the output of the function by the new length $d\sqrt{x}$, which looks like this.
+
+$$
+x + dx = (\sqrt{x} + d\sqrt{x})(\sqrt{x} + d\sqrt{x})
+$$
+
+To find the derivative, $\frac{d\sqrt{x}}{dx}$, expand out the area expressions on the right-hand side.
+
+$$
+x + dx = x + 2\sqrt{x}(d\sqrt{x}) + (d\sqrt{x})^2
+$$
+
+Since we are only interested in the change in area, we can subtract $x$ from both sides. Additionally, we can safely ignore the expression $(d\sqrt{x})^2$ since the area it represents changes the output (length) of the function in a negligible way.
+
+$$
+dx = 2\sqrt{x}(d\sqrt{x})
+$$
+
+Finally, we can divide both sides by $dx$ and $2\sqrt{x}$ to get the expression below. The derivative of $f(x) = \sqrt{x}$!
+
+$$
+\frac{d\sqrt{x}}{dx}  = \frac{1}{2\sqrt{x}}
+$$
+
+It's always a good idea to check our work, and luckily we have derived such a tool to do that, the power rule. Starting with $\displaystyle f(x) = x^{\frac{1}{2}}$ we can drop the $\frac{1}{2}$ down in front and subract one from the exponent, giving us the same answer.
+
+$$
+\frac{d\sqrt{x}}{dx}  = \frac{1}{2} x^{-\frac{1}{2}}
+$$
+
+</Question>
+
+### Volume of Sphere
+
+In the first chapter, we found a formula for the area of a circle using a small nudge to the radius of the circle. Importantly, we derived the formula by finding the area under a function. In future chapters, we will explore this idea much further and derivatives have a big role to play. For now, let's revisit the same idea using the geometry of a sphere.
+
+<Figure
+  image="figures/sphere.svg" 
+/>
+
+<Question
```

Comment:
We should actually move this whole question much later to the chapter on integrals. It's not as relevant here, I think the sqrt question would be a fine place to end it.

Reply:
Sounds great, I had a feeling it was a bit out of place.